api/src/pcapi/routes/backoffice_v3/templates/pro_user/get/details/history.html

```diff
@@ -20,8 +20,16 @@
                 <th scope="row"></th>
                 <td>{{ action.actionType.value }}</td>
                 <td>{{ action.actionDate | format_date("Le %d/%m/%Y à %Hh%M") }}</td>
-                <td class="px-4">
-                    <p>{{ action.comment | empty_string_if_null }}</p>
+                <td class="px-4 text-break">
+                    {% if action.actionType.name == "USER_INFO_MODIFIED" %}
+                        <p>Informations modifées :</p>
+                        {% for modified_data in action.extraData.get('modified_info', {}) %}
+                            <p>{{ modified_data | i18n_column_name }}: 
+                                {{ action.extraData.get('modified_info', {})[modified_data][0] | escape }} => {{ action.extraData.get('modified_info', {})[modified_data][1] | escape }}</p>
```

Comment:
C'est bizarre de refaire chaque fois `action.extraData.get('modified_info', {})`.
Peux-tu essayer plutôt avec : `for field_name, values in action.extraData.get('modified_info', {})items()` pour la boucle ?

Reply:
Aussi, le `>` devrait plutôt être `&gt;`, même si les navigateurs sont permissifs le `>` ferme une balise html